tomcat10/src/main/java/com/linecorp/armeria/server/tomcat/TomcatService.java

```diff
@@ -532,8 +532,7 @@ private Request convertRequest(ServiceRequestContext ctx, String mappedPath, Agg
         coyoteReq.protocol().setString(ctx.sessionProtocol().isMultiplex() ? "HTTP/2.0" : "HTTP/1.1");
 
         // Set the method.
-        final HttpMethod method = req.method();
-        coyoteReq.method().setString(method.name());
+        coyoteReq.method().setString(req.headers().get(HttpHeaderNames.METHOD));
```

Comment:
Can you also add a integration test which validates the server can successfully process the custom http method?